src/walletConnect/index.tsx

```diff
@@ -1002,13 +1006,22 @@ export async function onAuthRequest(event: Web3WalletTypes.AuthRequest) {
     }
   };
 
+  // need to prefetch dapp metadata since portal is static
+  const url =
+    // @ts-ignore Web3WalletTypes.AuthRequest type is missing VerifyContext
+    event?.verifyContext?.verifyUrl || event.params.requester.metadata.url;
```

Comment:
nit but i prefer `??` over `||` when optional

pretty sure they're functionally equivalent

Reply:
`??` is a lil more restrictive, here verifyUrl can be an empty string`""`

`||` will fallback
`??` will not fallback and use the empty string